third_party/terraform/utils/provider_handwritten_endpoint.go.erb

```diff
@@ -173,17 +173,6 @@ var RuntimeConfigCustomEndpointEntry = &schema.Schema{
 	}, RuntimeConfigDefaultBasePath),
 }
 
-var ServiceManagementDefaultBasePath = "https://servicemanagement.googleapis.com/v1/"
```

Comment:
Hmm I think we do need to leave these in until 4.0.0 so that anyone that was setting the custom endpoint  still can

Reply:
Oh gosh, I keep changing the product name to make the docs line up right but the product name also goes in here 🤦‍♂

Reply:
Fixed by changing the docs to use the product's display name and moving the product that produces this IAM resource to use the same namespace as the existing handwritten one.